package.json

```diff
@@ -24,7 +26,8 @@
     "name": "Stefan Rusu",
     "url": "http://www.saltwaterc.eu/"
   },
-  "contributors": [{
+  "contributors": [
```

Comment:
This won't survive the automatic formatting I'm afraid as everything is passed through js-beautify, including the json files.

Reply:
didn't meant to touch this